<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orderWindow</class>
 <widget class="QMainWindow" name="orderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="menuDrop">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="productLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:  $</string>
    </property>
   </widget>
   <widget class="QLabel" name="qtyLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity: </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="qtyBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalLine">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menuDrop</sender>
   <signal>activated(int)</signal>
   <receiver>orderWindow</receiver>
   <slot>setProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setProduct()</slot>
 </slots>
</ui>
